### PR TITLE
Kartikey Dubey made changes

### PR DESCRIPTION
**Pull request by Kartik125**
This is the pledge for Code for Cause

### CODE REVIEW

**Comment by Kartik125** [comment]
and I also added one new Avatar

**Comment by Kartik125** [comment]
waiting to merge

File: src/data/contributors/ sonuKushwaha.json [hunk 1]
```diff
@@ -1,8 +0,0 @@
-{
-  "name": "Sonu kumar Kushwaha",
-  "skills": ["Java", "HTML", "CSS"],
-  "cause": "learn and create new things",
-  "image": "Avatar5",
-  "github": "https://github.com/sonukushwaha403",
-  "linkedin": "https://www.linkedin.com/in/sonu-kushwaha-b0ab22190"
```
Comment (Abhishek-kumar09) [inline src/data/contributors/ sonuKushwaha.json:7]: Please revert these changes.

File: src/data/contributors/Kartikey.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Kartikey Dubey",
+  "skills": ["Java", "c", "C++","Python"],
+  "cause": "Learn and Explore new things",
+  "image": "Avatar7",
```
Comment (Abhishek-kumar09) [inline src/data/contributors/Kartikey.json:5]: Choose from the already created avatars. If you want to add a new Avatar then it should be a part of a new PR.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
Thanks for your PR, some minors.

File: src/data/contributors/Kartikey.json (hunk 2 above)
Comment (Kartik125) [reply src/data/contributors/Kartikey.json:5]: okay
